javascript.md

```diff
@@ -764,12 +849,63 @@ JavaScript:
 const data = {
   prop: "value"
 }
+console.log(data); // outputs Object { prop: "value" }
+console.log(data.prop); // outputs value
 ```
 
-Rust:
+Rust doesn't have a direct equivalent of this.
+
+If you want a proper object, you need to declare a type:
 
 ```rust
-// TODO
+#[derive(Debug, Clone)]
+pub struct Data {
+    prop: String,
+}
+
+let data = Data { prop: String::from("value") };
+println!("{:?}", data); // outputs Data { prop: "value" }
+println!("{:?}", data.prop); // outputs "value"
+```
+
+Most structs have private fields, but if you want client code to be able to access fields directly, you need to make them public:
+
+```rust
+#[derive(Debug, Clone)]
+pub struct Data {
+    pub prop: String,
+//  ^^^
+}
+```
+
+Or you can use a tuple, which is more ad-hoc:
```

Comment:
I like this use of a tuple better; it doesn't have an element in the tuple of the string "prop", so I feel like it's not implying that tuples have keys.